diesel_cli/src/print_schema.rs

```diff
@@ -280,12 +280,13 @@ impl Display for CustomTypeList {
                         writeln!(out, "/// (Automatically generated by Diesel.)")?;
                     }
                     if self.derives.is_empty() {
+                        writeln!(out, "#[derive(diesel::sql_types::SqlType)]")?;
```

Comment:
I think, it would be cleaner if `self.derives` would be augmented with `diesel::sql_types::SqlType` if that type not already presented there